Update Configuring a codespace for VS (#1416)
github/docs: content/github/developing-online-with-codespaces/using-codespaces-in-visual-studio.md

### Configuring a codespace for {% data variables.product.prodname_vs %}

A codespace, created with {% data variables.product.prodname_vs %}, can be customized through a new tool called devinit, a command line tool included with {% data variables.product.prodname_vs %}.

#### devinit

[devinit](https://docs.microsoft.com/visualstudio/devinit/getting-started-with-devinit) lets you install additional frameworks and tools into your Windows development codespaces, modify environment variables, and more. 

> devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json), which can be added to your project for creating customized and repeatable development environments. Used with a `.devcontainer.json`, your codespace will be automatically configured upon creation.

Reviewer: Small change to use active voice. 
I also wondered what  "Used with a `.devcontainer.json`" meant when I read this. It's explained in the page referenced in the sentence that follows this, but at this stage readers may not know. Perhaps link to the relevant section of that page here:

```suggestion
devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json). You can add this to your project if you want to create a customized and repeatable development environment. Used with a `.devcontainer.json`, your codespace will be automatically configured on creation.
```

https://docs.microsoft.com/en-us/visualstudio/ide/codespaces/customize-codespaces?view=vs-2019#running-devinit-when-creating-a-codespace

Author: Good suggestions. I pushed a commit with a few additional changes and a normalized URL.
